### ISSUE TITLE
[pip] provide additional requirements

### ISSUE THREAD

**Issue by TheInitializer** [Airget/airget-master#1]
When I installed airget I had to also install click and wget - could you bundle those with the main airget package? Thanks.

**Comment by Airget** [comment]
no, click and wget come with the setup.py, but that it tailored to mac
right now. it wouldnt work.
